test/functional/Valhalla/src/org/openj9/test/lworld/ValhallaAttributeTests.java

```diff
@@ -28,39 +28,65 @@
 @Test(groups = { "level.sanity" })
 public class ValhallaAttributeTests {
 
-	class MultiPreloadTest {}
+	static class MultiLoadableDescriptorsTest {}
 
-	/* A class may have no more than one Preload attribute. */
-	@Test(expectedExceptions = java.lang.ClassFormatError.class, expectedExceptionsMessageRegExp = ".*Multiple Preload attributes.*")
-	static public void testMultiplePreloadAttributes() throws Throwable {
-		String className = MultiPreloadTest.class.getName();
-		ValhallaAttributeGenerator.generateClassWithTwoPreloadAttributes("MultiPreloadAttributes",
-			new String[]{className}, new String[]{className});
+	/* A class may have no more than one LoadableDescriptors attribute. */
+	@Test(expectedExceptions = java.lang.ClassFormatError.class,
+		expectedExceptionsMessageRegExp = ".*Multiple LoadableDescriptors attributes.*")
+	static public void testMultipleLoadableDescriptorsAttributes() throws Throwable {
```

Comment:
Please list modifiers in canonical order (`public static void ...`) (at least those declarations added or modified by this change).

Reply:
This does not appear to have been addressed.